qemu/tests/block_disk_not_default_options.py

```diff
@@ -0,0 +1,114 @@
+from virttest import error_context, qemu_qtree
+
+from provider import disk_utils
+from provider.block_devices_plug import BlockDevicesPlug
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test the disk non-default options.
+
+    Steps:
+        1. Boot a VM with packed=true and page-per-vq=true
+        2. Verify the packed and page-per-vq values
+        3. Do some basic I/O operation in the VM
+        4. Unplug and hotplug the disk
+        5. Do again a basic I/O operation
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def verify_non_default_options(option_key, expected_value):
```

Comment:
if images have different options need be check, how to know the option is addressed on image?

Reply:
Ignore. so far, we just consider the single image option check, and serial is known in advance to simplify logic.